### PR TITLE
Make log and workspace dirs use the same timestamp in their names

### PR DESCRIPTION
**Pull request by bluebytes60**

### CODE REVIEW

File: build.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func grimBuild(builder grimBuilder, resultPath string) (*executeResult, string,
 	return result, workspacePath, nil
 }
 
-func build(token, configRoot, workspaceRoot, resultPath, clonePath, owner, repo, ref string, extraEnv []string) (*executeResult, string, error) {
+func build(token, configRoot, workspaceRoot, resultPath, clonePath, owner, repo, ref string, extraEnv []string, basename string) (*executeResult, string, error) {
 	ws := &workspaceBuilder{workspaceRoot, clonePath, token, configRoot, owner, repo, ref, extraEnv}
-	return grimBuild(ws, resultPath)
+	return grimBuild(ws, resultPath,basename)
```
Comment (bhand-mm) [inline build.go]: Please configure your editor to run `goimports` and `golint` on save.

Comment (bluebytes60) [inline build.go]: sure I am working on

File: workspace_test.go [hunk 2]
```diff
@@ -29,9 +31,9 @@ func TestPreparePublicRepo(t *testing.T) {
 		t.Errorf("|%v|", err)
 	}
 	defer os.RemoveAll(f)
-
+	basename := fmt.Sprintf("%v", time.Now().UnixNano())
```
Comment (kklipsch) [inline workspace_test.go]: duplicate of https://github.com/MediaMath/grim/pull/55/files#diff-9fe6a2c7832721bba3826ce78a216ebeR185

Should be a function.

File: github_archive_test.go [hunk 3]
```diff
@@ -18,8 +19,8 @@ func TestUnarchiveRepo(t *testing.T) {
 		t.Errorf("|%v|", err)
 	}
 	defer os.RemoveAll(f)
-
-	ws, err := createWorkspaceDirectory(f, "baz", "foo.bar")
+	basename := fmt.Sprintf("%v", time.Now().UnixNano())
```
Comment (kklipsch) [inline github_archive_test.go]: Duplicate of https://github.com/MediaMath/grim/pull/55/files#diff-9fe6a2c7832721bba3826ce78a216ebeR185

Should be a function.

File: Utils.go [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package grim
```
Comment (kklipsch) [inline Utils.go]: File should be named base_name.go

File: Utils.go [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+import (
+	"fmt"
+	"time"
+)
+
+//This function returns a timestamp
+func getTimeStamp() string {
```
Comment (kklipsch) [inline Utils.go]: createBaseName()

**Pull request merged**